docs/pages/enroll-resources/workload-identity/spiffe.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Introduction to SPIFFE
+description: Learn about Secure Production Identity Framework For Everyone (SPIFFE) and how it is implemented by Teleport Workload Identity
+---
+
+SPIFFE (Secure Production Identity Framework For Everyone) is a set of standards
+for securely identifying workloads.
+
+SPIFFE sets out:
+
+- A format for uniquely specifying an identity, the SPIFFE ID.
+- Standards for encoding the SPIFFE ID into verifiable documents, the SVID.
+- Processes that workloads should use to validate a received SVID.
+- A set of APIs that workloads can use to request SVIDS, the Workload API.
+
+The open nature and popularity of SPIFFE make it a great choice as a foundation
+for a full workload identity implementation. It is supported as an identity
+provider by a number of popular tools (such as Linkerd and Istio) and
+off-the-shelf SDKs exist for implementing SPIFFE directly into your own
+services.
+
+It's important to recognize that SPIFFE does not specify how to use SPIFFE IDs
+for authorization. This gives a high level of flexibility, allowing you to
+implement authorization in a way that suits you.
+
+## SPIFFE IDs and Trust Domains
```

Comment:
```suggestion
## SPIFFE IDs and trust domains
```
We are standardizing on [sentence case in headings](https://github.com/gravitational/docs/blob/main/docs-contributors/style-guide.md#headings).